uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/API/Query/LeaveRequestSelect.php

```diff
@@ -300,4 +301,14 @@ private function getToilLeaveRequests($result) {
 
     return $toilLeaveRequestIDs;
   }
+
+  /**
+   * Returns a list of relationship types stored on the
+   * 'relationship_types_allowed_to_approve_leave' setting
+   *
+   * @return array
+   */
+  private function getLeaveApproverRelationshipTypes() {
```

Comment:
Can we extract this function to a class seeing that the function is also duplicated in ACLQueryHelper.php class?
Will be easy for us to make changes if its in a central place.